### PR TITLE
ignore vs environment files

### PR DESCRIPTION
**Pull request by tigercosmos**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +13,9 @@ setup.mk
 *.pyo
 *.so
 *.dylib
+*.pyd
+*.vcx*
+*.sln
+*log
```
Comment (yungyuc) [inline .gitignore:19]: `*log` looks like a little bit wild.  Could it be more specific?

Comment (tigercosmos) [reply .gitignore:19]: VS creates a `modmesh.tlog` folder, please refer to this: https://stackoverflow.com/questions/41417471/what-are-these-tlog-files-should-these-be-part-of-my-source-control
I just think something like `debug.log`, `tmp.log.1` should also be ignored.

Comment (yungyuc) [reply .gitignore:19]: `*log` will also ignore a directory named like `syslog`.  I do not want that.

What about making it like `*.tlog`?  Additional ignoring entries can be added when we see them.

Comment (tigercosmos) [reply .gitignore:19]: sure

**Comment by tigercosmos** [comment]
I am closing this one, another PR will include this

**Comment by yungyuc** [comment]
No problem.  Just for your information.  I am fine to have small PRs as long as they make sense.  This one makes sense.

**Pull request closed without merging**